packages/date-picker/src/panel/date.vue

```diff
@@ -52,35 +52,41 @@
             <button
               type="button"
               @click="prevYear"
+              @keyup.enter="prevYear"
               :aria-label="t(`el.datepicker.prevYear`)"
               class="el-picker-panel__icon-btn el-date-picker__prev-btn el-icon-d-arrow-left">
             </button>
             <button
               type="button"
               @click="prevMonth"
+              @keyup.enter="prevMonth"
               v-show="currentView === 'date'"
               :aria-label="t(`el.datepicker.prevMonth`)"
               class="el-picker-panel__icon-btn el-date-picker__prev-btn el-icon-arrow-left">
             </button>
-            <span
+            <button
               @click="showYearPicker"
```

Comment:
use `click.stop`, do not add event param to {next, prev}{Year, Month} and show{Year, Month}Picker methods.